### DISCUSSION TITLE
Can't build this theme!

### DISCUSSION THREAD

**Discussion by Moal68** [nusserstudios/tailbliss#190] [Q&A] [answered]
npm run build

> TailBliss@0.1.1 build
> hugo --minify

Start building sites …
hugo v0.126.2-8f3d902ce51512931f5759e9279d93e346c9e649+extended windows/amd64 BuildDate=2024-05-30T15:19:22Z VendorInfo=gohugoio

Total in 156 ms
Error: error building site: "D:\Web\hugo\mypage\layouts\partials\posts-template.html:51:1": parse failed: template: partials/posts-template.html:51: bad number syntax: "2020-02"

Line 51 in layouts/partials/posts-template.html:
<time datetime="{{.Date.Format 2020-02-12}}">{{.Date.Format "2006-01-02"}}</time>

In my opinion there is a faulty syntax: 
The expression {{.Date.Format 2020-02-12}} is incorrect. In Hugo, {{.Date.Format}} is used to format a date according to a specific format. 
However, Hugo expects a string or a variable that represents a valid date here, not a direct numeric value like 2020-02-12.

So I used a valid date value variable like this:
<time datetime="{{.Date.Format "2006-01-02"}}">{{.Date.Format "2006-01-02"}}</time>

This works.

**Comment by nusserstudios** [comment, marked as answer]
Fixed, Thank you!
